int32 order_number
string order_type
string drink_type  # "americano" or "latte"
string temperature  # "ice" or "hot"
bool syrup  # True if syrup is added, False otherwise
int32 count
---
bool success
float64 execution_time
